packages/discovery-provider/integration_tests/tasks/entity_manager/test_user_entity_manager.py

```diff
@@ -430,6 +434,7 @@ def get_events_side_effect(_, tx_receipt):
         )
         assert user_2.name == "Forrest"
         assert user_2.handle == "forrest"
+        assert user_2.preferred_coin_flair_mint is None
```

Comment:
We're setting the mint in the create and then testing that we can delete it by passing `null`. The above test that sets empty string is sufficient to check that we index a valid string value correctly.